### ISSUE TITLE
Support for android code challenges

### ISSUE THREAD

**Issue by mohamedrez** [Codewars/codewars-runner-cli#424]
We are using codewars for java challenges and we are wondering if it is possible to have android support.

There is this project http://robolectric.org/ that de-fangs the Android SDK jar so you can test-drive the development of an Android app

**Comment by jhoffner** [comment]
I didn't know about roboelectric but that would be awesome to have this on Codewars/Qualified. This certainly seems to be possible. Once Java is moved over to Groovyserve (see #415) this should be even easier to implement. 

@mohamedrez any interest in trying to contribute support for this?

**Comment by mohamedrez** [comment]
Yes I am really interested in I will take a look to Groovyserve.

**Comment by mohamedrez** [comment]
Yes I want to contribute I am testing Roboelectric. I will update this thread once I found a solution
